From 184a3ce544389fa1f16188882cc01e8c9407b890 Mon Sep 17 00:00:00 2001
Message-Id: <184a3ce544389fa1f16188882cc01e8c9407b890.1350493760.git.minovotn@redhat.com>
In-Reply-To: <c93188b6c6973932d2adaa52e6a4920db13b4e62.1350493760.git.minovotn@redhat.com>
References: <c93188b6c6973932d2adaa52e6a4920db13b4e62.1350493760.git.minovotn@redhat.com>
From: Jeffrey Cody <jcody@redhat.com>
Date: Wed, 17 Oct 2012 05:59:23 +0200
Subject: [PATCH 10/35] block: New bdrv_get_flags()

RH-Author: Jeffrey Cody <jcody@redhat.com>
Message-id: <42741ccfb9544ef844a9e8e604e4940264cf98e2.1350447475.git.jcody@redhat.com>
Patchwork-id: 43269
O-Subject: [RHEL6.4 qemu-kvm PATCH v4 10/35] block: New bdrv_get_flags()
Bugzilla: 767233
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Eric Blake <eblake@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

From: Markus Armbruster <armbru@redhat.com>

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Acked-by: Paolo Bonzini <pbonzini@redhat.com>
Signed-off-by: Kevin Wolf <kwolf@redhat.com>
(cherry picked from commit c8433287830f342d8ee35d206afdf6cc6c5fe35a)
Signed-off-by: Jeff Cody <jcody@redhat.com>
---
 block.c | 5 +++++
 block.h | 1 +
 2 files changed, 6 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block.c | 5 +++++
 block.h | 1 +
 2 files changed, 6 insertions(+)

diff --git a/block.c b/block.c
index a2cc214..1f255c1 100644
--- a/block.c
+++ b/block.c
@@ -2104,6 +2104,11 @@ const char *bdrv_get_device_name(BlockDriverState *bs)
     return bs->device_name;
 }
 
+int bdrv_get_flags(BlockDriverState *bs)
+{
+    return bs->open_flags;
+}
+
 void bdrv_flush_all(void)
 {
     BlockDriverState *bs;
diff --git a/block.h b/block.h
index 8fe6708..52155bc 100644
--- a/block.h
+++ b/block.h
@@ -275,6 +275,7 @@ int bdrv_query_missing_keys(void);
 void bdrv_iterate_format(void (*it)(void *opaque, const char *name),
                          void *opaque);
 const char *bdrv_get_device_name(BlockDriverState *bs);
+int bdrv_get_flags(BlockDriverState *bs);
 int bdrv_write_compressed(BlockDriverState *bs, int64_t sector_num,
                           const uint8_t *buf, int nb_sectors);
 int bdrv_get_info(BlockDriverState *bs, BlockDriverInfo *bdi);
-- 
1.7.11.7

